pkgs/by-name/bu/buildstream/package.nix

```diff
@@ -21,28 +21,16 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "buildstream";
-  version = "2.5.0";
+  version = "2.6.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "apache";
     repo = "buildstream";
     tag = version;
-    hash = "sha256-/kGmAHx10//iVeqLXwcIWNI9FGIi0LlNJW+s6v0yU3Q=";
+    hash = "sha256-gEGAiywNJkAA3Nika3M74999uy7ORx8evdfitn/teic=";
```

Comment:
fyi I get the same hash mismatch - to build this locally I need to change:

```suggestion
    hash = "sha256-2Z+s0dQB85MBO06llhIEO3jwWfL53n74S28ENHcbe/Q=";
```

Reply:
:thinking: are you sure it's not a stale hash? It would be strange if @Sigmanificient and I both had the same transient error which produces the same hash. If you change one character of the hash does the build fail and suggest the hash you currently have?

Reply:
Weird. Tried changing one char of the hash, and I got what you and @Sigmanificient got. When I reverted the change, and re-ran the build, it still passed. Maybe my Nix store is broken.

Reply:
No, this is because the old hash makes nix reuse a content you previously had

Reply:
I've updated the hash to the correct one and re-pushed.